ui/page/dcrdex/dex_onboarding_page.go

```diff
@@ -965,6 +986,13 @@ func (pg *DEXOnboarding) HandleUserInteractions() {
 			}
 
 			pg.bondServer = serverInfo
+			if dexc.InitializedWithPassword() && len(pg.dexPass) == 0 {
```

Comment:
Is it possible to get here if dexc has not been initialized with password? If yes, how is that handled? If no, then `dexc.InitializedWithPassword()` doesn't need to be part of the if statement.

Reply:
> Is it possible to get here if dexc has not been initialized with password? If yes, how is that handled? 

Yes, a password is required for users to create a new dex instance from seed.  If this happens and the user navigates away from the page, we need to ask for the password because it's no longer cached.